CONTRIBUTING.md

```diff
@@ -61,3 +61,144 @@ To execute e2e tests run:
 ```
 go test test/e2e_test.go -v -count=1
 ```
+
+## Perf-test
+
+The performance test of metrics-server depends on the `clusterLoader` tool in `kubernetes/perf-tests` repository.
+
+`clusterLoader` supports prometheus to scrape metrics from metrics-server to achieve metrics-server latency measurement.
+
+### Design principle
+
+* metrics-server implements two APIs for performance testing, MetricsServerPrometheus and InClusterAPIServerRequestLatency.
+  The implementation of MetricsServerPrometheus is basically the same as that of kube-state-metrics, done in the prometheus framework of `clusterloader`.
+  The implementation of InClusterAPIServerRequestLatency, done in the probes about measurement framework of `clusterloader`.
+
+* According to the performance testing framework of `clusterLoader`, metrics-server implements two actions here
+
+    * `start` Complete the preparations for the start of the test
+
+      MetricsServerPrometheus: Record start time
+      InClusterAPIServerRequestLatency: Record start time and access the PodMetrics resource periodically.
+
+    * `gather`
+      Collect metrics-server metrics data through prometheus to achieve metrics server latency measurement
+
+###  Access Perf-test
+* Deploy kubernetes cluster
+
+
+* Clone perf-tests repository
+
+  Prepare the go environment
+  Start with cloning perf-tests repository:
+
+```
+  git clone git@github.com:kubernetes/perf-tests.git
+  cd perf-tests/clusterloader2
+```
+
+Note: the code needs to be cloned to the gopath directory.
+
+* Deploy metrics-server
+
+
+* Run metrics server latency measurement
+
+```
+./run-e2e.sh cluster-loader2 --provider=xxx --enable-prometheus-server=true  --prometheus-scrape-metrics-server=true --prometheus-apiserver-scrape-port=6443  --testconfig=metrics-server-scrape-config.yaml --prometheus-manifest-path=pkg/prometheus/manifests
```

Comment:
Since the tutorial is oriented toward kind, I think the `--provider` flag should be set to `kind` here, it would be easier for readers that just want to copy/paste the given commends.